Fashionz/assets/styles.css

```diff
@@ -0,0 +1,201 @@
+    text-decoration: none;
+    color: black;
+}
+
+/*Dropdown menu list hover CSS*/
+.fashionz-menu-dropdown a:hover  {
+    color: white;
+}
+/* ----------------- END :: Header ---------------------- */
+
+/* ----------------- START :: Main section ---------------------- */
+/* CSS for main container for each card */
+.main-section-container {
+    margin-bottom: 100px;
+}
+
+/* Star icon CSS */
+.star-icon {
+    position: absolute;
+    margin-top: 20px;
+    margin-left: 210px;
+    font-size: 20px;
+    color: #7ea479;
+    cursor: pointer;
+}
+
+/* Solid star icon on hover CSS */
+.solid-star-icon {
+    position: absolute;
+    margin-top: 20px;
+    margin-left: 210px;
+    font-size: 20px;
+    color: #7ea479;
+    z-index: -1;
+}
+
+.star-icon:hover ~.solid-star-icon {
+  z-index: 1;
+}
+
+/* Dress thumbnail card CSS */
+.dress-thumbnail {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin: 40px 20px 20px 20px;
+    width: 276;
+}
+
+/* Dress thumbnail container CSS */
+.dress-thubnail-container {
+    width: 250px;
+    height: 320px;
+    background-color: #24252a;
+}
+
+/* Dress img CSS */
+.dress-thubnail-container img {
+    margin: 70px 60px;
+}
+
+/* CSS for shirt size */
+.shirt-size {
+    width: 45px;
+    height: 45px;
+    border-radius: 50%;
+    position: absolute;
+    display: flex;
+    margin-top: -20px;
+    margin-left: 180px;
+    justify-content: center;
+    align-items: center;
+    background-color: #88b181;
+    font-size: 20px;
+    font-weight: 500;
+}
+
+/* Dress info - dress name CSS */
+.dress-name {
+    margin-top: 15px;
+    font-size: 18px;
+    font-weight: 200;
+    color: white;
+}
+
+/* Dress info - dress price CSS */
+.dress-price {
+    margin-top: 7px;
+    color: #aaaaaa;
+    font-weight: 500;
+}
+
+/* Dress info - Add to card button CSS */
+.add-to-cart-btn {
+    margin-top: 15px;
+    padding: 12px 12px;
+    width: max-content;
+    font-size: 13px;
+    background-color: #2a2b30;
+    color: #a4a4a4;
+    border: 1px solid;
+    cursor: pointer;
+}
+
+/* Dress info - Add to card button- hover CSS */
+.add-to-cart-btn:hover {
+    background-color: #88b181;
+    color: black
+}
+/* ----------------- END :: Main section ---------------------- */
+
+
+/* ----------------- START :: Fixed navbar ---------------------- */
+/* Bottom navbar CSS */
+.fashionz-navbar {
+    width: 100%;
+    bottom: 0px;
+    position: fixed;
+    font-size: 30px;
+    z-index: 1;
+    color: white;
+    text-align: center;
+    background-color: black;
+}
+
+/* Bottom navbar menu items CSS */
+.fashionz-navbar div{
+    padding: 30px 10px 10px 10px;
+    display: inline-block;
+    background-color: black;
+}
+
+/* Bottom navbar menu items link tag CSS */
+.fashionz-navbar div a{
+    text-decoration: none;
+    color: white;
+    background-color: black;
+}
+
+/* Bottom navbar menu items link tag hover CSS */
+.fashionz-navbar div:hover{
+    font-size: 32px;
```

Comment:
Increasing the font size on hover will lead to misalignments in structure until it is handled with a transition